cmd/agent/api/agent/agent.go

```diff
@@ -200,10 +201,14 @@ func getConfigCheck(w http.ResponseWriter, r *http.Request) {
 	var response response.ConfigCheckResponse
 
 	configs := common.AC.GetLoadedConfigs()
```

Comment:
Should `GetLoadedConfigs` returns a copy instead ?

Reply:
@ajacquemot yes I'll change this, there might be a race condition possible 🏎